source/sqlsrv/config.m4

```diff
@@ -29,14 +29,18 @@ if test "$PHP_SQLSRV" != "no"; then
     fi
         AC_MSG_RESULT($sqlsrv_inc_path)    
         
-  HOST_OS_ARCH=`uname`
-  if test "${HOST_OS_ARCH}" = "Darwin"; then
-      MACOSX_DEPLOYMENT_TARGET=`sw_vers -productVersion` 
-  fi
-  
   CXXFLAGS="$CXXFLAGS -std=c++11"
   CXXFLAGS="$CXXFLAGS -D_FORTIFY_SOURCE=2 -O2"
   CXXFLAGS="$CXXFLAGS -fstack-protector"
+
+  HOST_OS_ARCH=`uname`
+  if test "${HOST_OS_ARCH}" = "Darwin"; then
+      PDO_SQLSRV_SHARED_LIBADD="$PDO_SQLSRV_SHARED_LIBADD -Wl,-bind_at_load"
+      MACOSX_DEPLOYMENT_TARGET=`sw_vers -productVersion`
+  else
+      PDO_SQLSRV_SHARED_LIBADD="$PDO_SQLSRV_SHARED_LIBADD -Wl,-z,now"
+  fi
+
```

Comment:
Does this need to be done for SQLSRV config.m4? PDO_SQLSRV_SHARED_LIBADD isn't used.

Reply:
Typo, fixed.